sorbet/rbi/manual.rbi

```diff
@@ -0,0 +1,8 @@
+# typed: strict
+# This file contains types that were not found by tapioca
+
+class Hash
+  sig { returns(String) }
+  def to_yaml
```

Comment:
This is added by `psych`.
```
[1] pry(Packwerk::PackageSet)> {}.method(:to_yaml).source_location
=> ["/Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/3.0.0/psych/core_ext.rb", 12]
```
`/Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/3.0.0/psych/core_ext.rb`
```
# frozen_string_literal: true
class Object
  def self.yaml_tag url
    Psych.add_tag(url, self)
  end

  ###
  # call-seq: to_yaml(options = {})
  #
  # Convert an object to YAML.  See Psych.dump for more information on the
  # available +options+.
  def to_yaml options = {}
    Psych.dump self, options
  end
end

if defined?(::IRB)
  require 'psych/y'
end
```

I'd probably use `tapioca` to add this and other types, but I'm having trouble using `tapioca` in this repo:
```
~/workspace/packwerk - (ae-bump-typed-levels) $ bundle exec tapioca gem psych
bundler: failed to load command: tapioca (/Users/alex.evanczuk/.rbenv/versions/3.0.0/bin/tapioca)
/Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/3.0.0/rubygems/dependency.rb:309:in `to_specs': Could not find 'sorbet-static' (>= 0) - did find: [sorbet-static-0.5.6360-x86_64-linux,sorbet-static-0.5.6360-universal-darwin-20,sorbet-static-0.5.6360-universal-darwin-19,sorbet-static-0.5.6360-universal-darwin-18,sorbet-static-0.5.6360-universal-darwin-17,sorbet-static-0.5.6360-universal-darwin-16,sorbet-static-0.5.6360-universal-darwin-15,sorbet-static-0.5.6360-universal-darwin-14] (Gem::MissingSpecVersionError)
Checked in 'GEM_PATH=/Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0:/Users/alex.evanczuk/.gem/ruby/3.0.0' , execute `gem env` for more information
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/3.0.0/rubygems/dependency.rb:319:in `to_spec'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/3.0.0/rubygems/specification.rb:974:in `find_by_name'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/tapioca-0.4.19/lib/tapioca/compilers/sorbet.rb:10:in `<module:Sorbet>'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/tapioca-0.4.19/lib/tapioca/compilers/sorbet.rb:9:in `<module:Compilers>'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/tapioca-0.4.19/lib/tapioca/compilers/sorbet.rb:8:in `<module:Tapioca>'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/tapioca-0.4.19/lib/tapioca/compilers/sorbet.rb:7:in `<top (required)>'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/tapioca-0.4.19/lib/tapioca/internal.rb:15:in `require'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/tapioca-0.4.19/lib/tapioca/internal.rb:15:in `<top (required)>'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/tapioca-0.4.19/exe/tapioca:19:in `require_relative'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/tapioca-0.4.19/exe/tapioca:19:in `<top (required)>'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/bin/tapioca:23:in `load'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/bin/tapioca:23:in `<top (required)>'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `load'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `kernel_load'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:28:in `run'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/cli.rb:474:in `exec'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/cli.rb:30:in `dispatch'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/cli.rb:24:in `start'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/exe/bundle:49:in `block in <top (required)>'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/lib/bundler/friendly_errors.rb:128:in `with_friendly_errors'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.22/exe/bundle:37:in `<top (required)>'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/bin/bundle:23:in `load'
        from /Users/alex.evanczuk/.rbenv/versions/3.0.0/bin/bundle:23:in `<main>'
```

I've had this issue before in my own repos (I think it's related to having an M1 macbook) and I fixed it by bumping bundler, but that is a larger change.

Also happy to move this under `psych` in `sorbet/rbi/gems`, but I wasn't sure if we wanted to mix manual gem RBIs with those generated by tapioca.

Reply:
🤔

Reply:
Psych is stdlib, so no one actually requires it and also why Tapioca cannot generate anything for it. The RBI file for Psych is already bundled with Sorbet [here](https://github.com/sorbet/sorbet/blob/master/rbi/stdlib/psych.rbi) and for the correct reasons they have not include the extension to `Object` in that file (since it only gets registered if someone does a `require "yaml"`)

If you add an explicit dependency on `psych` in the Gemfile, Tapioca should be able to generate the proper extension to `Object`.

Reply:
Hmm, actually, I see that Tapioca does generate the `yaml_tag` method on the singleton class but not the `to_yaml` method for some reason.

Reply:
Strike that, that was a project that had both Syck and Psych, so Syck was generating the `to_yaml` method instead.

Here is one from a fresh project with `gem "psych"` in the `Gemfile`:
```ruby
class Object < ::BasicObject
  include ::Kernel

  def to_yaml(options = T.unsafe(nil)); end

  class << self
    def yaml_tag(url); end
  end
end
```